contrib/babelfishpg_tsql/src/linked_servers.c

```diff
@@ -873,7 +875,9 @@ getOpenqueryTupdescFromMetadata(char *linked_server, char *query, TupleDesc *tup
 		StringInfoData buf;
 		int			colcount;
 
-		linked_server_establish_connection(linked_server, &lsproc);
+		bool isTesting = false;
```

Comment:
no need to define a bool variable for this. Call the linked_server_establish_connection() function directly with third argument as false